wsgi.py

```diff
@@ -0,0 +1,4 @@
+from customersupport import app
+
+if __name__ == "__main__":
+    app.run(debug=True)
```

Comment:
Is this running in `debug` mode? If not, I think we could just remove that section.